Fix Android Sign in with Google command (#2956)
clerkinc/clerk-docs: docs/guides/configure/auth-strategies/sign-in-with-google.mdx

     - **Package name**: Your package name is in your `build.gradle` file, formatted as `com.example.myclerkapp`.
     - **SHA-1 certificate fingerprint**: To get your SHA-1, run the following command in your terminal:

       > [!IMPORTANT]

>         > Replace `path-to-debug-or-production-keystore` with the path to your debug or production keystore. By default, the debug keystore is located in `~/.android/debug.keystore`. It may ask for a keystore password, which is `android`. **You may need to install [OpenJDK](https://openjdk.org/) to run the `keytool` command. Also note that Java is required to run this command.**

Reviewer: ```suggestion
       > [!IMPORTANT]
        > Replace `path-to-debug-or-production-keystore` with the path to your debug or production keystore. By default, the debug keystore is located in `~/.android/debug.keystore`. It may ask for a keystore password, which is `android`. **You may need to install [OpenJDK](https://openjdk.org/) to run the `keytool` command.**
```
